index.php

```diff
@@ -0,0 +1,95 @@
+<html lang='en'>
+    <head>
+        <link rel="stylesheet" href="css.css">
+        <link href="https://fonts.googleapis.com/css?family=Baloo+2&display=swap" rel="stylesheet">
+        <title>Road Bike Collection</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body onTouchStart="">
+        <h1>Collection of Road Bikes</h1>
+        <div class="collectionContainer1">
+            <div class="collectionItem">
+                <h2>Collection Item 1</h2>
+                <img class="collectionImage" src="https://dbyvw4eroffpi.cloudfront.net/product-media/2L9M/1000/1000/Cannondale-SystemSix-Carbon-Ultegra-Di2-Disc-Road-Bike-2020.jpg"></img>
+                <ul class="informationList">
+                    <li>Name:</li>
+                    <li>Brand:</li>
+                    <li>Price:</li>
+                    <li>Class:</li>
+                </ul>
+            </div>
```

Comment:
Eventually I assume you'd be generating this from the database using PHP

Reply:
correct, its placeholder for now